src/domain/utils/hash.ts

```diff
@@ -12,11 +12,11 @@ class InvalidHashingAlgorithm extends Error {
 }
 
 export const hashSync = (data: string | Uint8Array, alg: string) => {
-  if (alg === Alg.SHA256) {
+  if (alg === Alg.SHA256 || alg === Alg.SHA256.toLocaleLowerCase()) {
     return Uint8Array.from(Hashing.sha256().update(data).digest());
   }
 
-  if (alg === Alg.SHA512) {
+  if (alg === Alg.SHA512 || alg === Alg.SHA512.toLocaleLowerCase()) {
```

Comment:
what case is this needed for? I was under the impression `alg` is a specific string?